client/app/admin/admin.component.ts

```diff
@@ -0,0 +1,154 @@
+import { Component, OnInit } from '@angular/core';
+
+import { AdminService } from './admin.service';
+import { MenuService } from '../menu/menu.service';
+
+@Component({
+    selector: 'app-admin',
+    templateUrl: './admin.component.html',
+    styleUrls: ['./admin.component.css']
+})
+export class AdminComponent implements OnInit {
+
+    public menu = [];
+
+    public staff = [
+        {
+            firstname: 'Klaus',
+            surname: 'Logan',
+            role: 'Waiter'
+        },
+        {
+            firstname: 'Susanne',
+            surname: 'Smith',
+            role: 'Waiter'
+        },
+        {
+            firstname: 'Joanne',
+            surname: 'Lee',
+            role: 'Manager'
+        },
+        {
+            firstname: 'John',
+            surname: 'McGregor',
+            role: 'Manager'
+        },
+        {
+            firstname: 'Lucy',
+            surname: 'McCabe',
+            role: 'Chef'
+        }
+    ];
+
+    private displayOrdersTodayBool;
+    private displayOrdersWeekBool;
+    private displayOrdersMonthBool;
+
+    constructor(
+        protected adminService: AdminService,
+        private menuService: MenuService
+    ) { }
+
+    ngOnInit() {
+        this.getMenu();
+    }
+
+    getMenu() {
+        this.menuService.getMenu().subscribe(result => {
+            this.menu = result;
+        })
+    }
+
+    updateMenu(itemId, menuItem) {
+        //this.menuService.update(itemId, menuItem);
+    }
+
+    addItem() {
+        var itemName = this.removeBrackets(document.getElementById("itemName").innerHTML);
+        var itemPrice = this.removeBrackets(document.getElementById("itemPrice").innerHTML);
+        var itemCategory = document.getElementById("menuCategorySelect").value;
+        console.log(itemName);
+        console.log(itemPrice);
+        console.log(itemCategory);
+        if (this.isNumeric(itemPrice)) {
+            var newItem = { name: itemName, price: itemPrice, category: itemCategory };
+            this.menuService.create(newItem);
+            this.getMenu();
+        } else {
+            alert("Item price is not a valid number. Try again and add to menu.");
+        }
+    }
+
+    removeBrackets(string) {
+        var replaced = string.replace(/\<[^\>]*\>/, '');
+        return replaced;
+    }
+
+    isNumeric(num) {
+        console.log(!isNaN(num));
+        return !isNaN(num);
+    }
+
+    openMenuNav() {
```

Comment:
Not sure this is the best way to update the css for those ids... but it works for now so we should maybe focus on getting all the functionality working end to end first and then improve the code.